js/gallery.js

```diff
@@ -0,0 +1,11 @@
+import {
+  getSimilarPosts
+} from './post-list.js';
+
+const getGalleryPosts = getSimilarPosts();
```

Comment:
```suggestion
const galleryPosts = getSimilarPosts();
```

а вообще не принято таким образом данные передавать